scripts/Game/Systems/Core/Managers/CRF_GamemodeManager.c

```diff
@@ -137,12 +139,28 @@ class CRF_GamemodeManager : SCR_BaseGameModeComponent
 	override void OnPostInit(IEntity owner)
 	{	
 		super.OnPostInit(owner);
-		
 		// Initialize all required manager references
 		InitializeManagers();
 		LoadConfigurations();
 	}
 	
+	//Needed so when we teleport players/vehicles the aren't spawning on top of each other.
+	float m_fBuffer = 0;
+	override void EOnFrame(IEntity owner, float timeSlice)
+	{
+		super.EOnFrame(owner, timeSlice);
```

Comment:
Also, every time we use an EOnFrame, instead of doing frame logic for the rest of the entity lifecycle, we should make sure it has at least one early exit. IE, if safestart is disabled in this case or something similiar. We don't want to be doing math or really any of this logic, for the entire game.